wire-ios-data-model/Source/Utilis/EAR/EARKeyRepository.swift

```diff
@@ -90,10 +90,10 @@ public class EARKeyRepository: EARKeyRepositoryInterface {
             keyCache[description.id] = key
             return key
         } catch KeychainManager.Error.failedToFetchItemFromKeychain(errSecItemNotFound) {
-            WireLogger.ear.warn("private key not found in keychain")
+            WireLogger.ear.warn("private key not found in keychain", attributes: .safePublic)
             throw EarKeyRepositoryFailure.keyNotFound
         } catch {
-            WireLogger.ear.warn("failed to fetch private key: \(error)")
+            WireLogger.ear.warn("failed to fetch private key: \(error)", attributes: .safePublic)
```

Comment:
Not sure here if the error could contain sensible information?

Reply:
probably not, it's ok